pyaerocom/io/readungridded.py

```diff
@@ -52,14 +56,15 @@ class ReadUngridded:
         ReadAeronetSunV3,
         ReadEarlinet,
         ReadEbas,
-        ReadGAW,
         ReadAasEtal,
-        ReadGhost,
         ReadAirNow,
```

Comment:
It's not clear to me why only GAW, Ghost, and MSCW CTM have their own plugins. Is there something special about these or do we just need to begin moving the other readers into this format later?

Reply:
These were easy to port. Other readers are more complicated.
We can port more readers on subsequent PRs.